PolyEngine/RenderingDevice/OpenGL/CMakeLists.txt

```diff
@@ -29,10 +29,24 @@ set(POLYGLDEVICE_H_FOR_IDE
 	Src/Text2DRenderingPass.hpp
 	Src/TransparentRenderingPass.hpp
 )
+set(POLYGLDEVICE_LIBS polycore polyengine epoxy::gl)
+
+# macOS Obj-C sources
+if(MACOSX)
+	set(POLYGLDEVICE_SRCS
+			${POLYGLDEVICE_SRCS}
+            Src-macOS/CocoaImpl.mm)
```

Comment:
Using `list(APPEND)` to, well, append elements to the list is probably the saner choice here

```
list(APPEND POLYGLDEVICE_SRCS 
	Src-macOS/CocoaImpl.mm
	anotherEntry.ext
)
```